code/game/objects/items/devices/scanners.dm

```diff
@@ -172,7 +172,7 @@ MASS SPECTROMETER
 
 	// Time of death
 	if(M.tod && (M.stat == DEAD || (M.status_flags & FAKEDEATH)))
-		user << "<span class='info'>Time of Death:</span> [M.tod]"
+		user << "<span class='info'>Time of Death: [M.tod]</span>"
 

```

Comment:
The time itself was a black text and everything else was neatly blue. You think that's intentional?